meshapp/src/components/login-form.tsx

```diff
@@ -111,8 +133,8 @@ const LoginScreen = (props: ComponentProps) => {
 }
 
 const ForgotPasswordScreen = (props: ComponentProps) => {
-
-  const forgotPasswordEndpoint: string = '/user/reset';
+  
+  const forgotPasswordEndpoint: string = 'http://localhost:8000/accounts/auth/users/reset_password/';
```

Comment:
Small nitpick, but might make more sense to just have a string that has the URL domain, and then just have the endpoint appended to that URL so that we can just change the URL domain string in the future.

Reply:
There actually isn't a need for the URL domain at all since front and backend are ran on the same port. For example, look at my LoginScreen component.

I plan on fixing this in my next PR. Since it involves making the forgot password form fully functional.